joomla-extensions/weblinks
# Move lang files for search plugin

Pull Request for Issue #290

### Summary of Changes

Move language files for the search plugin into the plugin distribution file. This makes all the plugins consistent in not putting language files into administrator/language.

### Testing Instructions

- Install 3.7.0b2
- apply this patch
- confirm the language files in the plugins folder

### Expected result

the language files in the plugins folder

### Actual result

the language files are not in the plugins folder

### Documentation Changes Required

None
